plugins/recommendations-plugin/package.json

```diff
@@ -19,7 +19,8 @@
     "globby": "^11.0.1",
     "inversify": "^5.0.1",
     "kind-of": "^6.0.3",
-    "reflect-metadata": "^0.1.13"
+    "reflect-metadata": "^0.1.13",
```

Comment:
I'm wondering how this is related to webpack 4 -> webpack 5. Maybe it's an independent problem and should be left for another PR.

Reply:
I'm trying to understand it either.
The problem https://github.com/eclipse-che/che-theia/pull/1134#pullrequestreview-681505810 is not reproduced on Che-Theia:next but on this PR.